wordpress/wp-content/plugins/memberful-wp/src/contrib/ad-providers/base-ad-provider.php

```
@@ -0,0 +1,155 @@
+   * Get the identifier of the ad provider.
+   *
+   * @return string The identifier of the ad provider.
+   */
+  public function get_identifier() {}
+
+  /**
+   * Disable ads for a user.
+   *
+   * @param int $user_id The ID of the user to disable ads for.
+   */
+  public function disable_ads_for_user($user_id) {}
```

PHPMD (2.15.0)

Avoid unused parameters such as '$user_id'. (undefined)

(UnusedFormalParameter)